source/whats-new.txt

```diff
@@ -0,0 +1,168 @@
+.. _cpp-whats-new:
+
+==========
+What's New
+==========
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: update, new feature, deprecation, upgrade, driver
+
+Overview
+--------
+
+Learn what's new in:
+
+* :ref:`Version 3.10 <version-3.10>`
+* :ref:`Version 3.9 <version-3.9>`
+
+Breaking Change Types
+~~~~~~~~~~~~~~~~~~~~~
+
+New versions of the {+driver-short+} can introduce the following types of breaking changes:
+following groups:
+
+- **API** changes concern the API of the {+driver-short+} libraries. These changes are
+  accompanied by an API major version bump. You might need to edit C++ source code
+  in your project to address these changes.
+
+  .. example::
+
+     - Removed function ``func(A)``. Use ``func(A, B)`` instead.
+     - Changed redeclaration of ``bsoncxx::foo`` from ``v1::foo`` to ``v2::foo``. Qualify
+       ``bsoncxx::foo`` as ``bsoncxx::v1::foo`` to continue using the old interface.
```

Comment:
```suggestion
     - Removes the declaration of function ``v1::func()``. (The definition is still provided for ABI compatibility.)
     - Removes the inline function `v_noabi::func()`.
     - Removes `v_noabi::func()`.
     - Changes the redeclaration of ``bsoncxx::foo`` from ``v1::foo`` to ``v2::foo``.
       - To continue using the old interface, qualify
       ``bsoncxx::foo`` as ``bsoncxx::v1::foo``.
```

Suggested wording highlights examples of API-only breaking changes.

Note: the "Removes `v_noabi::func()`" case demonstrates the library exercising its right to make ABI breaking changes to entities declared under `v_noabi`, whose ABI stability is not supported. Therefore the removal of such a symbol could be documented under API breaking changes without an accompanying ABI breaking change entry. If this is confusing, we can also list the corresponding ABI breaking change with "Removes the exported symbol `v_noabi::func()`." as suggested in another comment below. See also the suggestion to add a note regarding the exemption of `v_noabi` symbols from ABI version bumps.